model/task_lifecycle.go

```diff
@@ -1185,9 +1185,12 @@ func MarkStart(t *task.Task, updates *StatusChanges) error {
 	return nil
 }
 
-func MarkTaskUndispatched(t *task.Task) error {
+// MarkHostTaskUndispatched marks a task as no longer dispatched to a host. If
+// it's part of a display task, update the display task as necessary as a result
```

Comment:
Does this mean that the caller should update the display task as necessary? Or does this function do take care of that? Wording is a bit confusing.

Reply:
It calls [this function](https://github.com/Kimchelly/evergreen/blob/33f5245e7f852c9b7c6dde6c85631025ada493db/model/task_lifecycle.go#L1462-L1569) to do a bunch of display task metadata updates that result from marking the task undispatched. I simplified the wording a bit.

Reply:
Thanks!